### PR TITLE
Add tests for moving secretaries registers

### PR DESCRIPTION
**Pull request by danfalc**
- Conditional test for filing EH03/EW03 depending on the current location
  of the registers.
- A test for checking accesibility of adding a SAIL address and the error
  fields
- Includes fixes to the move directors registers journey

### CODE REVIEW

File: cypress/integration/location_of_company_records/change_location_of_secretaries_register.spec.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+import ManageRegistersPreFilingPage from '../../support/page_objects/ManageRegistersPreFilingPage'
+import ChangeSecretariesRegisterLocationPage from '../../support/page_objects/ChangeSecretariesRegisterLocationPage'
+import BasePage from '../../support/page_objects/generic/BasePage'
+
+const companyOverview = new CompanyOverviewPage();
+const allFormsPage = new AllFormsPage();
+const manageRegistersPreFilingPage = new ManageRegistersPreFilingPage();
+const submissionConfirmationPage = new SubmissionConfirmationPage();
+const changeSecretariesRegisterLocationPage = new ChangeSecretariesRegisterLocationPage();
+const basePage = new BasePage();
```
Comment (rsuller) [inline cypress/integration/location_of_company_records/change_location_of_secretaries_register.spec.js]: Unused and should only really be extended from other page object classes.

File: cypress/integration/location_of_company_records/change_location_of_secretaries_register.spec.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+describe('EH03/EW03 - Elect to Hold Secretaries Register on the Public Register or at Registered Office Address', () => {
+    beforeEach(function () {
```
Comment (rsuller) [inline cypress/integration/location_of_company_records/change_location_of_secretaries_register.spec.js]: I am not sure if we should opt with the other version over `function` as it does allow us to provide a description which could be useful e.g.
```
'Select secretaries register', () =>
```

File: cypress/support/page_objects/ChangeSecretariesRegisterLocationPage.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+// Constants 
+const invalidCharacter = "`";
+
+class ChangeSecretariesRegisterLocationPage {
```
Comment (rsuller) [inline cypress/support/page_objects/ChangeSecretariesRegisterLocationPage.js]: How much code is duplicated here between this and `ChangeDirectorsRegisterLocationPage.js`?
Is it worth extracting the duplication to one class?

Comment (danfalc) [reply cypress/support/page_objects/ChangeSecretariesRegisterLocationPage.js]: One final registers test to go. EW02/EW03. Can maybe look at this while doing that today?

File: cypress/integration/location_of_company_records/change_location_of_secretaries_register.spec.js (hunk 2 above)
Comment (danfalc) [reply cypress/integration/location_of_company_records/change_location_of_secretaries_register.spec.js]: Agreed

File: cypress/integration/location_of_company_records/change_location_of_secretaries_register.spec.js (hunk 1 above)
Comment (danfalc) [reply cypress/integration/location_of_company_records/change_location_of_secretaries_register.spec.js]: 👍 copy and paste. Festive week making me lazy!!

File: cypress/support/page_objects/ChangeSecretariesRegisterLocationPage.js (hunk 3 above)
Comment (rsuller) [reply cypress/support/page_objects/ChangeSecretariesRegisterLocationPage.js]: Yeah that seems fair 👍

File: cypress/integration/location_of_company_records/change_location_of_secretaries_register.spec.js (hunk 1 above)
Comment (rsuller) [reply cypress/integration/location_of_company_records/change_location_of_secretaries_register.spec.js]: 😂

**Pull request merged**